src/practise/Collections/AppendElementToHashSet.java

```diff
@@ -0,0 +1,20 @@
+package Git.src.practise.Collections;
+
+import java.util.HashSet;
+import java.util.Scanner;
+
+public class AppendElementToHashSet {
+    public static void main(String[] args) {
+        HashSet<Integer> set = new HashSet<>();
+        set.add(1);
+        set.add(2);
+        set.add(4);
+        Scanner scanner = new Scanner(System.in);
+        int element = scanner.nextInt();
+        if (element > 4) {
```

Comment:
HashSet stores the element in the index of the remainder after dividing with 10. Hence when we give small number it will not be stored at the end. Hence giving any number Greater than 4 will append at the last.